hardhat.config.ts

```diff
@@ -68,7 +79,8 @@ const config: NilHardhatUserConfig = {
     },
   },
   walletAddress: process.env.WALLET_ADDR,
-  directTxGasLimit: 500000
+  debug: true,
+  shardId: 1,
```

Comment:
shard is already default to 1, we don't need to set

Reply:
agree